.gitignore

```diff
@@ -25,3 +25,6 @@ package-lock.json
 
 # Generated by storybook@4
 .cache
+
+# Example build artifacts
+.next
```

Comment:
Could we move this to a local `.gitignore` by the example? Like: `examples/next/.gitignore`?

Reply:
Totally!